parity-util-mem/CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
 # Changelog
 
-The format is based on [Keep a Changelog]. 
+The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [0.4.0] - 2020-01-01
+- Added implementation of `MallocSizeOf` for non-std `hashbrown::HashMap` and `lru::LRUMap`. [#293](https://github.com/paritytech/parity-common/pull/293)
```

Comment:
I think #300 is also worth mentioning (Added support for tuples with length up to 12)

Reply:
More like an implementation detail to me, but ok :)